packages/@aws-cdk/aws-lambda/lib/lambda-version.ts

```diff
@@ -223,6 +246,36 @@ export class Version extends QualifiedFunctionBase implements IVersion {
     return addAlias(this, this, aliasName, options);
   }
 
+  /**
+   * Enable autoscaling for the given lambda version. Requires that lambda version has provisioned concurrency.
+   * @param props The properties for autoscaling
+   */
+  public autoScaleProvisionedConcurrency(props: EnableScalingProps): IScalableVersionAttribute {
+    if (!this.provisionedConcurrency) {
+      throw new Error('Autoscaling is available for versions with provisioned concurrency only');
+    }
+    if (this.scalableVersion) {
+      throw new Error('Autoscaling already enabled for this version');
+    }
+    // Use a Service Linked Role
+    // https://docs.aws.amazon.com/autoscaling/application/userguide/application-auto-scaling-service-linked-roles.html
+    const role = iam.Role.fromRoleArn(this, 'ScalingRole', Stack.of(this).formatArn({
+      service: 'iam',
+      region: '',
```

Comment:
though there is no region for iam, how does formatArn know which partition it's in without region?